src/vector/feature.rs

```diff
@@ -648,12 +648,28 @@ impl<'a> Feature<'a> {
         }
     }
 
+    /// Clear a field, marking it as null.
+    ///
+    /// See: [`OGRFeature::SetFieldNull`][SetFieldNull]
+    ///
+    /// [SetFieldNull]: https://gdal.org/api/ogrfeature_cpp.html#_CPPv4N10OGRFeature12SetFieldNullEi
     pub fn set_field_null(&self, field_name: &str) -> Result<()> {
         let idx = self.field_idx_from_name(field_name)?;
         unsafe { gdal_sys::OGR_F_SetFieldNull(self.c_feature(), idx) };
         Ok(())
     }
 
+    /// Clear a field, marking it as unset.
+    ///
+    /// See: [`OGRFeature::UnsetField`][UnsetField]
+    ///
+    /// [UnsetField]: https://gdal.org/api/ogrfeature_cpp.html#_CPPv4N10OGRFeature10UnsetFieldEi
+    pub fn unset_field(&self, field_name: &str) -> Result<()> {
```

Comment:
I guess that should be (same for set_field_null())
```suggestion
    pub fn unset_field(&mut self, field_name: &str) -> Result<()> {
```

Reply:
It should, but all of these currently take `&self` for some reason and @metasim ran into some issues when trying to change them to `&mut self`.

Reply:
That's https://github.com/georust/gdal/issues/499. I think it should work, but didn't try it yet.